models/articles.models.js

```diff
@@ -17,22 +18,31 @@ exports.selectArticle = async (id) => {
     return article;
 }
 
+exports.selectAllArticles = async () => {
+    
+    const query = `SELECT articles.*, COUNT(comment_id)::INTEGER AS comment_count FROM articles
+                    INNER JOIN comments ON comments.article_id = articles.article_id
```

Comment:
is inner join getting back all of the articles or only the articles that have comments?